sharing_configs/admin.py

```diff
@@ -170,19 +168,27 @@ def import_from_view(self, request, **kwargs):
             self.model._meta.app_label,
             self.model._meta.model_name,
         )
+        main_url = f"admin:{info[0]}_{info[1]}_import"
+        ajax_url = f"admin:{info[0]}_{info[1]}_ajax"
+        context = Context()
+        context.update({"main_url": main_url, "ajax_url": ajax_url})
```

Comment:
```suggestion
        import_urls = {"main_url": main_url, "ajax_url": ajax_url}
```

Please see my remark from my previous PR: this dict is added to the `context` below, so let's call it something else than `context`

Reply:
Done